### PR TITLE
test confs

### PR DESCRIPTION
**Pull request by loechel**

### CODE REVIEW

**Comment by loechel** [comment]
@tkimnguyen could you review please and merge

@pavithirakc this pull request is adding automated tests to this package. 
All Test should always pass for a pull request

you could start the test by calling the tox command

File: buildout.cfg [hunk 1]
```diff
@@ -62,14 +62,6 @@ eggs = i18ndude
 # Don't use a released version of ploneorg.addonlisting
 ploneorg.addonlisting =
 
-setuptools = 24.3.0
```
Comment (tkimnguyen) [inline buildout.cfg:65]: These pins are no longer needed?

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+
+
+# -- Options for Texinfo output -------------------------------------------
+
+# Grouping the document tree into Texinfo files. List of tuples
+# (source start file, target name, title, author,
+#  dir menu entry, description, category)
+texinfo_documents = [
+    (master_doc, 'ploneorg.addonlisting', u'ploneorg.addonlisting Documentation',
+     author, 'ploneorg.addonlisting', 'One line description of project.',
```
Comment (tkimnguyen) [inline docs/conf.py]: Maybe update the description

File: src/ploneorg/addonlisting/browser/views.py [hunk 3]
```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 
-from Products.Five.browser import BrowserView
-from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
-
+from plone.protect.interfaces import IDisableCSRFProtection
 from ploneorg.addonlisting.utils import update_addon
 from ploneorg.addonlisting.utils import update_addon_list
-from plone.protect.interfaces import IDisableCSRFProtection
```
Comment (tkimnguyen) [inline src/ploneorg/addonlisting/browser/views.py:8]: always glad to see this not be used :)

File: src/ploneorg/addonlisting/vocabularies.py [hunk 4]
```diff
@@ -42,21 +43,32 @@ def addon_type_vocabulary(context):
 @provider(IVocabularyFactory)
 def addon_pypi_state_vocabulary(context):
     return SimpleVocabulary([
-        SimpleTerm(value=u'status-1', title=u'Development Status :: 1 - Planning'),
```
Comment (tkimnguyen) [inline src/ploneorg/addonlisting/vocabularies.py]: my opinion: it's more legible and less error-prone to have these terms on just one line, even if it violates a hard code formatting standard

**Review by tkimnguyen — COMMENTED** [review]
Thank you! I actually feel not worthy to be reviewing your PRs :)

File: src/ploneorg/addonlisting/browser/views.py (hunk 3 above)
Comment (loechel) [reply src/ploneorg/addonlisting/browser/views.py:8]: yes should be removed later, but now I just do add automated tests and make the tests green for flake8 and isort errors.

@pavithirakc that is in there for first debug in the browser views. It should be removed in the final product.

File: buildout.cfg (hunk 1 above)
Comment (loechel) [reply buildout.cfg:65]: Those Pins are not necessary or did even harm the tests.

As this is not the production buildout, that does not matter much.

**Comment by loechel** [comment]
@tkimnguyen I do like the Idea to not merge everything right by myself. A Review is always good.
Everybody could learn from a code review.

File: src/ploneorg/addonlisting/vocabularies.py (hunk 4 above)
Comment (loechel) [reply src/ploneorg/addonlisting/vocabularies.py]: should be applied.

File: docs/conf.py (hunk 2 above)
Comment (loechel) [reply docs/conf.py]: done

**Review by tkimnguyen — APPROVED** [review]
thx for accepting my comments with grace :)

**Pull request merged**